docs/getting_started/use-cases.rst

```diff
@@ -196,12 +196,12 @@ Adding Options
 --------------
 
 By default, some of the commands will come with singularity options. For example,
-a container intended for gpu is always going to give you the ``--nv`` flag. However,
+a container intended for gpu should have the ``--nv`` flag. However,
```

Comment:
How about this?
```suggestion
a container intended for gpu will have a feature: gpu set to true, and this will add an the ``--nv`` flag given that the user or cluster settings file has that feature enabled. However,
```

Reply:
Fine with me!